tasks/setup-Debian.yml

```diff
@@ -18,19 +18,20 @@
 - name: Download specific Jenkins version.
   get_url:
     url: "{{ jenkins_pkg_url }}/jenkins_{{ jenkins_version }}_all.deb"
-    dest: "/tmp/jenkins.deb"
+    dest: "/tmp/jenkins_{{ jenkins_version }}.deb"
   when: jenkins_version is defined
 
 - name: Check if we downloaded a specific version of Jenkins.
   stat:
-    path: "/tmp/jenkins.deb"
+    path: "/tmp/jenkins_{{ jenkins_version }}.deb"
+  when: jenkins_version is defined
```

Comment:
I think this `when` condition should be removed (similar to the RedHat issue below).